powerauth-webflow/src/main/java/io/getlime/security/powerauth/app/webflow/exception/DefaultExceptionResolver.java

```diff
@@ -120,4 +119,19 @@ public String handleUnauthorizedException(InsufficientAuthenticationException ex
         final Error error = new Error(Error.Code.ERROR_GENERIC, ex.getMessageId());
         return new ErrorResponse(error);
     }
+
+    /**
+     * Handling of invalid token exception.
+     * @param ex Exception.
+     * @return Response with error details.
+     */
+    @ExceptionHandler(InvalidTokenException.class)
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    public @ResponseBody ErrorResponse handleInvalidTokenException(InvalidTokenException ex) {
+        logger.warn("Error occurred in Web Flow server: {}", ex.getMessage());
+        audit.warn("Error occurred in Web Flow server", AUDIT_DETAIL_BAD_REQUEST, ex);
```

Comment:
I see the whole exception in audit, but I am thinking about logging it at `debug` level too.
```suggestion
        logger.warn("Error occurred in Web Flow server: {}", ex.getMessage());
        logger.debug("Error occurred in Web Flow server", ex);
        audit.warn("Error occurred in Web Flow server", AUDIT_DETAIL_BAD_REQUEST, ex);
```

Reply:
We'd have to change this in all exception handlers for consistency, let's handle such changes separately.